includes/admin/class-wc-admin-addons.php

```diff
@@ -316,7 +316,7 @@ public static function output_wcs_banner_block( $block = null ) {
 		$location  = wc_get_base_location();
 
 		if (
-			'US' !== $location['country'] ||
+			! in_array( $location['country'], array( 'US', 'CA' ) ) ||
```

Comment:
If we are going to show this to store owners in Canada, then we should update the description copy. Right now, it says:

> WooCommerce Services makes shipping a breeze. Print a label, take advantage of discounted shipping rates, and send tracking information to your customer right as you process your order, all from the convenience of your WordPress dashboard.

It's not currently possible to print a label, or use discounted rates in Canada. Only possible to show live rates.